vortex-fastlanes/src/for/compress.rs

```diff
@@ -29,6 +30,11 @@ impl EncodingCompression for FoREncoding {
             return None;
         }
 
+        // For all-null, cannot encode.
```

Comment:
without this, arrays that have all-nulls would fail on the `compute_as_cast` a few lines below